src/utility/util.ts

```diff
@@ -357,6 +363,52 @@ export class Util {
     }
     return true;
   }
+  public static async deleteDownloadedLesson(
+    lessonId: string
+  ): Promise<boolean> {
+    try {
+      if (!Capacitor.isNativePlatform()) return true;
+
+      await Filesystem.rmdir({
+        path: `${lessonId}`,
+        directory: Directory.External,
+        recursive: true,
+      });
+      console.log("lessonIDD", lessonId);
+    } catch (error) {
+      console.log("Error deleting lesson:", error);
+    }
+    return true;
+  }
+
+  public static async checkDownloadedLessons() {
```

Comment:
Don't check it every time the user opens the app; check after some time interval